docs/style_guide.md

```diff
@@ -123,3 +121,28 @@ if (length(foo()) == 1) {
 3. Avoid inline CSS styles.
 4. In CSS file, each rule should be on its own line.
 5. Avoid using percentages for widths; instead, use bootstrap's grid system by specifying columns.
+
+## General package code
+1. Create file _{pkgname}-package.R_ with `usethis::use_package_doc()`
+2. Update _{pkgname}-package.R_ - it should have such lines:
+```
+#' @note To learn more about functions start with `help(package = "{pkgname}")` 
+#' @keywords internal
+#' @return package help page
+"_PACKAGE"
+```
+3. Add in the DESCRIPTION `Roxygen: list(markdown = TRUE)`
+4. All constants, imports and side-effects tools should be in file _package.R_. Do not use _zzz.R_
```

Comment:
Following conversation with @gladkia and @bczech  I would recommend adding separate section about imports (maybe here?). It could contain following information:
* If some functions/packages are often used within the package we always add @import or @importFrom in one place, (_ package.R_). 
* When using function from another package use "pkg::function_name".

Reply:
I have added that as a sub point to 4. as it is content of _package.R_